### PR TITLE
prep-html-script-tag exercise- Kayla Cloud

### PR DESCRIPTION
**Pull request by kaylacloud**
Screenshot 2024-01-09 at 1 41 48 PM
Screenshot 2024-01-09 at 4 41 38 PM

### CODE REVIEW

File: prep-html-script-tag/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <title>  HTML Script Tag </title>
+  </head>
+  <body>
+    <script> console.log ("I have added JS to a script tag in the HTML!") </script>
+    <script src="main.js"></script>
+    <script> console.log ("I have added a script file via the src attribute!") </script>
```
Comment (BrettAlbright) [inline prep-html-script-tag/index.html]: This console.log should be in the main.js file.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
one small update needed for completion!

**Comment by kaylacloud** [comment]
Screenshot 2024-01-09 at 4 41 38 PM

File: prep-html-script-tag/main.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML script Tag </title>
+  </head>
+  <body>
+console.log("I have added a script file via the src attribute!")
```
Comment (BrettAlbright) [inline prep-html-script-tag/main.js]: only the console log should be in this file. Js files don't have HTML in them

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
One more update!

**Comment by kaylacloud** [comment]
Screenshot 2024-01-11 at 4 14 46 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**